composer.json

```diff
@@ -25,15 +25,13 @@
         }
     },
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.2",
-        "illuminate/support": "^5.0",
-        "nesbot/carbon": "~1.21",
-        "predis/predis": "^1.1"
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.4",
+        "nesbot/carbon": "^2.27"
```

Comment:
`illuminate/support` is needed for the Facades

Reply:
Ok i will add it